dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 cmake
 pytest
-scikit-build
+-e git://github.com/SylvainCorlay/scikit-build.git@filter-manifest#egg=scikit-build
```

Comment:
Requires feature branch for PR https://github.com/scikit-build/scikit-build/pull/476.